crates/rpc-types-engine/Cargo.toml

```diff
@@ -15,18 +15,26 @@ exclude.workspace = true
 workspace = true
 
 [dependencies]
+snap = "1.1.1"
```

Comment:
[snap](https://github.com/BurntSushi/rust-snappy) is licensed under the `BSD-3 Clause`. We need to include this license to publish a crate with this dep I believe.

Reply:
This should be as simple as adding [the license](https://github.com/BurntSushi/rust-snappy/blob/master/COPYING) as a new file in the repo root and referencing it in the module level doc comment in `payload.rs`.